static/src/stylesheets/module/_adslot.scss

```diff
@@ -252,11 +247,7 @@
         clear: left;
     }
 }
-.ad-slot__content {
-    > div {
-        margin: 0 auto;
-    }
-}
+
```

Comment:
This rule is far too broad - depending on what kind of advert is being rendered, DFP can add in a bunch of different nested divs, which this rule eagerly adds margins to.